### ISSUE TITLE
flex reentrant yylineno uninitialised

### ISSUE THREAD

**Issue by srivasta** [westes/flex#60]
Hi,

  This bug was reported by a Debian user at https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=616099

If you ask for a reentrant scanner with yylineno, nothing initialises yylineno for you.  This is not clearly documented AFAICT.

So you have to explictly call yyset_lineno.  I think it would be convenient if yylex_init did this for you.  Ie, yy_init_globals should clear yylineno.

Manoj
[libxlu_cfg_l.l.txt](https://github.com/westes/flex/files/145828/libxlu_cfg_l.l.txt)

**Comment by westes** [comment]
Yes, it's correct that yylineno should be set. Thanks for the report.
